### PR TITLE
Update CONTRIBUTING.md and add READMEs to every module

### PR DESCRIPTION
**Pull request by maximAtanasov**

### CODE REVIEW

File: coderadar-test/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# coderadar-test
+ 
+This module contains all of the integration tests. Unit tests are however contained withing the individual modules.
```
Comment (KilianKrause) [inline coderadar-test/README.md:3]:
```suggestion
This module contains all of the integration tests. Unit tests are however contained within the individual modules.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -68,16 +75,17 @@ is made up of the following elements:
 ### Building coderadar
 Simply run `gradlew build`.
 
-### Starting the coderadar server
-* install a local database (MySQL is preconfigured, so if you want to reduce configuration time, use MySQL)
-* copy the file `coderadar-server/local.application.properties.template` into a file named `local.application.properties`
+### Starting coderadar
+* install Neo4j
+* copy the file `coderadar-app/src/main/resources/local.application.properties.template` into a file named `local.application.properties`
```
Comment (KilianKrause) [inline CONTRIBUTING.md:80]:
```suggestion
* copy the file `coderadar-app/src/main/resources/application.properties.template` into a file named `application.properties`
```

Comment (maximAtanasov) [reply CONTRIBUTING.md]: Fixed

File: coderadar-test/README.md (hunk 1 above)
Comment (maximAtanasov) [reply coderadar-test/README.md]: Fixed

**Pull request merged**